insonmnia/miner/plugin/config.go

```diff
@@ -1,11 +1,12 @@
 package plugin
 
 type Config struct {
-	SocketPath string        `yaml:"path"`
-	Volumes    VolumesConfig `yaml:"volume"`
+	SocketDir string                       `yaml:"socket_dir" default:"/run/docker/plugins"`
+	Volumes   VolumesConfig                `yaml:"volume"`
+	GPUs      map[string]map[string]string `yaml:"GPUs"`
```

Comment:
No need to speficy attribute here. Moreover in yaml it is default to use lowercase names.